daemon/algod/api/server/v2/test/handlers_test.go

```diff
@@ -338,6 +338,34 @@ func addBlockHelper(t *testing.T) (v2.Handlers, echo.Context, *httptest.Response
 	return handler, c, rec, stx, releasefunc
 }
 
+func TestGetBlockTxids(t *testing.T) {
+	partitiontest.PartitionTest(t)
+	t.Parallel()
+
+	handler, c, rec, stx, releasefunc := addBlockHelper(t)
+	defer releasefunc()
+
+	err := handler.GetBlockTxids(c, 0)
+	require.NoError(t, err)
+	require.Equal(t, 200, rec.Code)
+
+	c, rec = newReq(t)
+	err = handler.GetBlockTxids(c, 2)
+	require.NoError(t, err)
+	require.Equal(t, 404, rec.Code)
+
+	c, rec = newReq(t)
+	err = handler.GetBlockTxids(c, 1)
+	require.NoError(t, err)
+
+	var response model.BlockTxidsResponse
+	data := rec.Body.Bytes()
+	err = protocol.DecodeJSON(data, &response)
+	require.NoError(t, err)
+	require.Equal(t, 1, len(response.BlockTxids))
```

Comment:
It's a shame there's only one txn in `addBlockHelper()`'s block. I see it's only used in 2 other tests so it could be modded. But I won't hold up the PR over this.

Reply:
Good point, I left it as-is in this PR but added a E2E test like you suggested [61064a4](https://github.com/algorand/go-algorand/pull/5611/commits/61064a43c38d185840f2f77d51496c6faeb98cc0)